### PR TITLE
Added custom inputAccessoryView

### PR DESCRIPTION
**Pull request by patricks**

### CODE REVIEW

File: Major Key/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13529" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13770" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
```
Comment (KrauseFx) [inline Major Key/Base.lproj/Main.storyboard:2]: That's a great change in particular, thanks Xcode 👍

File: Major Key/MajorKeyView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+//
+
+import UIKit
+
+protocol MajorKeyViewDelegate: class {
+    func didPressKeyButton(button: UIButton)
+    func didPressPackageButton(button: UIButton)
+}
+
+class MajorKeyView: UIView {
```
Comment (KrauseFx) [inline Major Key/MajorKeyView.swift:16]: What is this? Proper iOS architecture and stuff? hehe

**Review by KrauseFx — APPROVED** [review]
That's awesome, now we actually have proper iOS code, hehe :) I'll test it locally before merging

**Comment by KrauseFx** [comment]
screenshot 2017-11-30 05 25 21

Mh, doesn't seem to compile for me, but also didn't take a look at it yet. Have you committed all the files?

**Comment by patricks** [comment]
Sorry my bad didn't commit the updated project file. Should build now. 👍

File: Major Key/MajorKeyView.swift (hunk 2 above)
Comment (patricks) [reply Major Key/MajorKeyView.swift:16]: What do you mean? This is the new custom class which handles the 2 buttons in the inputAccessoryView.

**Comment by KrauseFx** [comment]
Thanks for the update, I just gave it a shot, and seems like the 📦  button now doesn't work any more? :)

**Comment by patricks** [comment]
Are you sure that problem is caused by this PR? I just removed the outlets and call the functions in the delegate callback. 

🔑: -> didPressMajorKey()
📦: -> didPressMajorKey() + didTapOldKeys()

**Comment by KrauseFx** [comment]
Ha, you're right! Thanks 👍

**Comment by KrauseFx** [comment]
Thanks for your contribution 👍

**Pull request merged**